proto/mixer.proto

```diff
@@ -190,13 +190,15 @@ message ObsTimeSeries {
     string measurement_method = 3;
     string observation_period = 4;
     string import_name = 5;
+    string provenanceDomain = 6;
```

Comment:
This deviates from the use of snake case above.

Reply:
changed.